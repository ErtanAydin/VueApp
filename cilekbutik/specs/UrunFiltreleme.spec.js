import { mount, RouterLinkStub } from "@vue/test-utils";
import UrunFiltreleme from "@/components/UrunFiltreleme";

describe("UrunFiltreleme", () => {
  test("mounts properly", () => {
    const wrapper = UrunFiltreleme;
    expect(wrapper).toBeTruthy();
  });

  test("renders properly", () => {
    const wrapper = mount(UrunFiltreleme, { 
      stubs: {
        NuxtLink: RouterLinkStub
      }
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});

describe ('data', () => {
  test('data test', () => {
    const wrapper = mount(UrunFiltreleme,{
    stubs: {
      NuxtLink: RouterLinkStub
    }});
    expect(wrapper.vm.AltKategoriler).not.toBe(undefined)
    expect(wrapper.vm.FiyatAralÄ±k).not.toBe(undefined)
  })
})