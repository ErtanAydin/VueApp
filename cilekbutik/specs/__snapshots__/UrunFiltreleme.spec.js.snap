// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UrunFiltreleme renders properly 1`] = `
<section class="UrunFiltrelemeSection">
  <div class="AltKategorilerDiv">
    <h3 class="AltKategorilerBaşlık">Alt Kategoriler</h3> <a title="Sweatshirt" class="AltKategoriler">Sweatshirt</a><a title="Sweatshirt" class="AltKategoriler">Gömlek/Bluz</a><a title="Sweatshirt" class="AltKategoriler">Crop</a> <span><br></span>
  </div>
  <div class="Markalar">
    <h3 class="AltKategorilerBaşlık">Markalar</h3> <input type="checkbox" class="UrunFiltrelemeCheckBox"> <span class="SpanImage"></span> <a href="#" class="CheckboxIsim"> Çilekbutik</a>
  </div>
  <div class="FiyatAralıkları">
    <h3 class="AltKategorilerBaşlık">Fiyat Aralıkları</h3>
    <div class="FiyatAralığı">
      <div><input type="radio" name="fiyat" class="UrunFiltrelemeCheckBox"> <span class="SpanImage"></span> <a href="#" class="CheckboxIsim"> 20 TL ve altı</a></div>
      <div><input type="radio" name="fiyat" class="UrunFiltrelemeCheckBox"> <span class="SpanImage"></span> <a href="#" class="CheckboxIsim"> 20 TL - 39 TL</a></div>
      <div><input type="radio" name="fiyat" class="UrunFiltrelemeCheckBox"> <span class="SpanImage"></span> <a href="#" class="CheckboxIsim"> 39 TL - 59 TL</a></div>
      <div><input type="radio" name="fiyat" class="UrunFiltrelemeCheckBox"> <span class="SpanImage"></span> <a href="#" class="CheckboxIsim"> 59 TL - 79 TL</a></div>
      <div><input type="radio" name="fiyat" class="UrunFiltrelemeCheckBox"> <span class="SpanImage"></span> <a href="#" class="CheckboxIsim"> 79 TL - 99 TL</a></div>
      <div><input type="radio" name="fiyat" class="UrunFiltrelemeCheckBox"> <span class="SpanImage"></span> <a href="#" class="CheckboxIsim"> 99 TL - 119 TL</a></div>
      <div><input type="radio" name="fiyat" class="UrunFiltrelemeCheckBox"> <span class="SpanImage"></span> <a href="#" class="CheckboxIsim"> 119 TL - 139 TL</a></div>
      <div><input type="radio" name="fiyat" class="UrunFiltrelemeCheckBox"> <span class="SpanImage"></span> <a href="#" class="CheckboxIsim"> 139 TL - 159 TL</a></div>
      <div><input type="radio" name="fiyat" class="UrunFiltrelemeCheckBox"> <span class="SpanImage"></span> <a href="#" class="CheckboxIsim"> 159 TL - 179 TL</a></div>
      <div><input type="radio" name="fiyat" class="UrunFiltrelemeCheckBox"> <span class="SpanImage"></span> <a href="#" class="CheckboxIsim"> 179 TL ve üzeri</a></div>
    </div>
  </div>
  <div class="Listeleme">
    <h3 class="AltKategorilerBaşlık">Listeleme</h3> <input type="checkbox" class="UrunFiltrelemeCheckBox"> <span class="SpanImage"></span> <a href="#" class="CheckboxIsim"> Sadece stoktakileri göster</a>
  </div> <button class="SeçimTemizleButon"> Seçimleri Temizle</button>
</section>
`;
